dawn/test/integration-test/dawn4py-tests/global_var.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+##===-----------------------------------------------------------------------------*- Python -*-===##
+##                          _
+##                         | |
+##                       __| | __ ___      ___ ___
+##                      / _` |/ _` \ \ /\ / / '_  |
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT).
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+import argparse
+import os
+
+import dawn4py
+from dawn4py.serialization import SIR
+from dawn4py.serialization import utils as sir_utils
+
+OUTPUT_NAME = "global_var_stencil"
+OUTPUT_FILE = f"{OUTPUT_NAME}.cpp"
+OUTPUT_PATH = f"{OUTPUT_NAME}.cpp"
+
+
+def main(args: argparse.Namespace):
```

Comment:
This test is not run (missing from `CMakeLists.txt`).
Since this test targets a feature of codegen i'd suggest making it a `VERIFY` one (need `data/global_var_ref.cpp`).

Reply:
(otherwise it's just testing that CodeGen doesn't crash)